daprdocs/content/en/operations/components/certification-lifecycle.md

```diff
@@ -0,0 +1,94 @@
+---
+type: docs
+title: "Certification lifecycle"
+linkTitle: "Certification lifecycle"
+weight: 200
+description: "The certification lifecycle of the different components available in Dapr"
+---
+
+## Overview
+
+Dapr uses a modular design where functionality is delivered as a component. Each component has an interface definition.  All of the components are pluggable so that in ideal scenarios, you can swap out one component with the same interface for another. Each component that is used in production, needs to matain a certain set of technical requirements that ensure the functional compatibility and robustness of the component.
+
+In general a component needs to be: 
+- compliant with the defined Dapr interfaces 
+- functionally correct and robust
+- well documented and maintained
+
+To make sure a component conforms to the standards set by Dapr, there are a set of tests run against a component in a Dapr maintainers managed environment. Once the tests pass consistently for a certain number of runs, the maturity level can be determined for a component. 
+
+## Certification levels 
+
+The proposed levels are as follows:
+- [Alpha](#alpha)
+- [Beta](#beta)
+- [General availability (GA)](#general-availability-ga)
+
+### Alpha
+
+- Component implements the required interface and works as described in the specification
+- Component has documentation
+- Component might be buggy or might expose bugs on integration
+- Component may not pass all conformance tests
+- Component may not have conformance tests
+- Not recommended for use in production
+
+All components generally start at the Alpha stage.
+
+### Beta
+
+- Component must pass all the component conformance tests defined to satisfy the component specification
+- The component conformance tests have been run in a Dapr maintainers managed environment
+- Component contains a record of the conformance test result reviewed and approved by Dapr maintainers with specific components-contrib version
+- Not recommended for use in production
```

Comment:
I would reword this to: `Recommended for only non-business-critical uses because of potential for incompatible changes in subsequent releases.`. The text is taken from kubernetes.io, I would rephrase a little.